test/Htseq_count/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_workflow {
+    
+
+    name "Test worfklow with Htseqcount"
+    script "../../main.nf"
+    workflow "RNA_seq"
+
+    stage {
+        symlink "/exports/sascstudent/vperinbanathan"
```

Comment:
Doesn't this mean these tests will only run on your system? It would be better if anyone could run them without having to adjust the test definitions.